crates/stages/src/stages/hashing_account.rs

```diff
@@ -15,7 +15,7 @@ use std::{
 use tracing::*;
 
 /// The [`StageId`] of the account hashing stage.
-pub const ACCOUNT_HASHING: StageId = StageId("AccountHashing");
+pub const ACCOUNT_HASHING: StageId = StageId("AccountHashingStage");
```

Comment:
This is unrelated, please remove it from the PR

Reply:
This is a side effect of a rebase. I'll remove it